pkg/api/frontendsettings.go

```diff
@@ -134,18 +134,19 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 		}
 
 		panels[panel.ID] = plugins.PanelDTO{
-			ID:            panel.ID,
-			Name:          panel.Name,
-			AliasIDs:      panel.AliasIDs,
-			Info:          panel.Info,
-			Module:        panel.Module,
-			BaseURL:       panel.BaseURL,
-			SkipDataQuery: panel.SkipDataQuery,
-			HideFromList:  panel.HideFromList,
-			ReleaseState:  string(panel.State),
-			Signature:     string(panel.Signature),
-			Sort:          getPanelSort(panel.ID),
-			Angular:       panel.Angular,
+			ID:              panel.ID,
+			Name:            panel.Name,
+			AliasIDs:        panel.AliasIDs,
+			Info:            panel.Info,
+			Module:          panel.Module,
+			BaseURL:         panel.BaseURL,
+			SkipDataQuery:   panel.SkipDataQuery,
+			PreferredHeight: panel.PreferredHeight,
```

Comment:
Is this being set in the `plugin.json`? And if so does that mean that once a panel plugin sets this, regardless of the datasource that's being used, the panel will be that specific height in explore?

Reply:
yes this is in the `plugin.json`. It would indeed mean that it will have the height regardless of the datasource. I am not sure this is too much of an issue. As the panel would be designed to display data in a specific way, so the source of that data should not affect (too much) the rendering.